src/description.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var s = require('util').format;
+var marked = require('marked');
```

Comment:
Since there's no security risk involved, it's the users responsibility to pass a proper markdown file or string if they want it to display on their docs... We can't really secure any possible distorted usages.

> `readFile`, if error, use description as is (string), otherwise use file contents

If we use `readFile` for file or string testing it means hitting the file system in both situations. Maybe not a big performance concern but still, feels a bit strange.
If there's an ENOENT, it could also be that the path is wrong (really common). In this case the path will be displayed on the theme...